.github/workflows/build_and_test_cmake.yml

```
@@ -47,14 +47,15 @@ jobs:
           LLVM_COMMIT=$(grep LLVM_COMMIT ${GITHUB_WORKSPACE}/bazel/import_llvm.bzl | head -n 1 | cut -d'"' -f 2 )
           git submodule update --init --recursive
           cd externals/llvm-project
+          git fetch
           git checkout ${LLVM_COMMIT}
           mkdir build && cd build
           cmake -G Ninja ../llvm -DLLVM_ENABLE_PROJECTS=mlir -DLLVM_BUILD_EXAMPLES=ON -DLLVM_ENABLE_ASSERTIONS=ON -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_RTTI=ON -DLLVM_TARGETS_TO_BUILD="host"
           cmake --build . --target check-mlir
 
       - name: Build and test mlir-tutorial
         run: |
-          mkdir build && cd build
+          mkdir build-cmake && cd build-cmake
```

This one's not needed, because the bazel build uses a randomly generated directory name somewhere else in the filesystem.

```suggestion
          mkdir build && cd build
```